src/util/ConfigManager/ConfigManager.h

```diff
@@ -275,8 +278,8 @@ class ConfigManager {
     move constructor. Which is why, we can't just return the `ConfigOption`
     we created here.
     */
-    return *configurationOptions_.at(createJsonPointerString(pathToOption))
-                .get();
+    return ConstConfigOptionProxy<OptionType>(
+        *configurationOptions_.at(createJsonPointerString(pathToOption)).get());
```

Comment:
If the (internal) `addConfigOption` function above already returns a `const ConfigOption&`, the you can also get rid of the redundant lookup via the key here.

Reply:
It doesn't.

Reply:
Not really the branch for this, but done. See [91d5117](https://github.com/ad-freiburg/qlever/pull/1055/commits/91d511774b25d124e938c1f657eac30bc44de597).